wgpu-core/src/device/mod.rs

```diff
@@ -2899,7 +2968,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                     Some(ref trace) => {
                         let mut trace = trace.lock();
                         let data = trace.make_binary("bin", unsafe {
-                            slice::from_raw_parts(ptr.as_ptr(), buffer.size as usize)
+                            std::slice::from_raw_parts(ptr.as_ptr(), buffer.size as usize)
```

Comment:
what happened here?

Reply:
The slice import goes unused when compiling with no features, so I removed it and fully qualified the couple of uses.